recipes/r-arrapply/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set version = '2.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-arrapply
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/arrApply_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/arrApply/arrApply_{{ version }}.tar.gz
+  sha256: 671c1e53b418191950cf6ddad56ef28d61dd1f367b69e6f570390354228fc763
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+    - {{posix}}sed               # [win]
+    - {{posix}}coreutils         # [win]
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-rcpp >=0.12.0
+    - r-rcpparmadillo
+    - libopenblas
```

Comment:
Please read https://conda-forge.org/docs/maintainer/knowledge_base.html#blas on how to specify blas correctly.

Reply:
Thanks @isuruf for precise doc pointer